
In here are the input and output files from the first (shared) run on the ARMS-MBON sequences
* Data are the raw sequences from material samples taken in 2018-2020. An overview of the material samples, ENA accession numbers, and names of the input parameter.tsv files used, can be found in the file (PEMAOverview.xlsx)[https://github.com/arms-mbon/data_workspace/blob/main/PEMAoutputs/Processing_001/PEMAOverview.xlsx]
* Processing used (PEMA version 2.1.4)[https://github.com/hariszaf/pema/tree/ARMS]. The version number can also be found in the parameter.tsv files that are in the folder (ParameterFiles)[https://github.com/arms-mbon/data_workspace/tree/main/PEMAoutputs/Processing_001/ParameterFiles]. These parameter files contain the name of the gene type and the date of processing 
* The ASV/OTU tables are the particular output files that contain the taxonomic assignments, and these can be found in (TaxonomicAssignments)[https://github.com/arms-mbon/data_workspace/tree/main/PEMAoutputs/Processing_001/TaxonomicAssignments]
* The cleaned, trimmed, and grouped sequences coming out of PEMA - fasta files - are generally too large to store here in GitHub: instead we have placed them in an external archive and links to them are given in the file (XXX)[[https://github.com/arms-mbon/data_workspace/tree/main/PEMAoutputs/Processing_001/XXX]. 

